docs/2-Presets/Laravel.md

```diff
@@ -97,6 +97,16 @@ PostgreSQL 13.0
 +DB_PORT=5432
 ```
 
+> In order to avoid permission issues with mysql, add a user other than root and a password to your **.env.example** file
```

Comment:
I'm not sure if this is an issue that we can face only with mysql, so I guess this text can be more generic.

Reply:
Just confirmed on the other presets available, it only happens on MySQL and mariaDB. I've updated the docs.